pkg/crc/preflight/preflight_darwin.go

```diff
@@ -39,13 +39,23 @@ var dnsPreflightChecks = [...]PreflightCheck{
 	},
 }
 
+var traySetupCheck = PreflightCheck{
+	configKeySuffix:  "check-tray-setup",
+	checkDescription: "Checking if the tray is installed and running",
+	check:            checkTrayExistsAndRunning,
+	fixDescription:   "Installing and setting up tray app",
+	fix:              fixTrayExistsAndRunning,
+	flags:            SetupOnly,
+}
+
```

Comment:
My feeling is that this preflight check is doing too much at the same time, we usually have smaller tests, one checking it's installed, another checking it's running, ...
This probably can be adjusted later on.

Reply:
I was thinking about the same, but then this was done to make the setup process be skip-able by configure just one config option.

Reply:
with the "experimental feature" thing gerard added, this could now be split if desired.